.github/workflows/performance-tests.yml

```diff
@@ -0,0 +1,21 @@
+name: Autocomment atime-based performance regression analysis on PRs
+
+on:
+  pull_request:
+    branches:
+      - '*'
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    container: ghcr.io/iterative/cml:0-dvc2-base1
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      repo_token: ${{ secrets.GITHUB_TOKEN }}
+      R_KEEP_PKG_SOURCE: yes
```

Comment:
I would add a quick comment on why this is needed as it's not immediately obvious

Reply:
I think I had it in my initial 'use my action from the marketplace' workflow for debugging purposes, but it's not required otherwise so I'll just omit it